Managers/UInputMan.cpp

```diff
@@ -2684,125 +2684,84 @@ int UInputMan::Update()
+		if (KeyHeld(KEY_2)) { g_TimerMan.SetTimeScale(g_TimerMan.GetTimeScale() + 0.01F); }
+		if (KeyHeld(KEY_1) && g_TimerMan.GetTimeScale() - 0.01F > 0.001F) { g_TimerMan.SetTimeScale(g_TimerMan.GetTimeScale() - 0.01F); }
 
-    // Only allow performance tweaking if showing the stats
-    if (g_PerformanceMan.IsShowingPerformanceStats())
-    {
-        // Manipulate timescaling
-        if (KeyHeld(KEY_2))
-            g_TimerMan.SetTimeScale(g_TimerMan.GetTimeScale() + 0.01);
-        if (KeyHeld(KEY_1) && g_TimerMan.GetTimeScale() - 0.01 > 0.001)
-            g_TimerMan.SetTimeScale(g_TimerMan.GetTimeScale() - 0.01);
-
-        // Increase real to sim cap
-        if (KeyHeld(KEY_4))
-            g_TimerMan.SetRealToSimCap(g_TimerMan.GetRealToSimCap() + 0.001);
-        // Decrease frame delay
-        if (KeyHeld(KEY_3) && g_TimerMan.GetRealToSimCap() > 0)
-            g_TimerMan.SetRealToSimCap(g_TimerMan.GetRealToSimCap() - 0.001);
-        // Manipulate deltatime
-        if (KeyHeld(KEY_6))
-            g_TimerMan.SetDeltaTimeSecs(g_TimerMan.GetDeltaTimeSecs() + 0.001);
-        // Decrease frame delay
-        if (KeyHeld(KEY_5) && g_TimerMan.GetDeltaTimeSecs() > 0)
-            g_TimerMan.SetDeltaTimeSecs(g_TimerMan.GetDeltaTimeSecs() - 0.001);
-    }
+		// Manipulate real to sim cap
+		if (KeyHeld(KEY_4)) { g_TimerMan.SetRealToSimCap(g_TimerMan.GetRealToSimCap() + 0.001F); }
+		if (KeyHeld(KEY_3) && g_TimerMan.GetRealToSimCap() > 0) { g_TimerMan.SetRealToSimCap(g_TimerMan.GetRealToSimCap() - 0.001F); }
 
-    // Screendump - Ctrl+S
-    if ((((key_shifts & KB_CTRL_FLAG)/* && (key_shifts & KB_SHIFT_FLAG)*/ && KeyHeld(KEY_S)) || KeyHeld(KEY_PRTSCR)))// && g_TimerMan.DrawnSimUpdate())
-    {
-        g_FrameMan.SaveScreenToBMP("ScreenDump");
-// TEMP!!
-//        g_FrameMan.SaveBitmapToBMP(g_SceneMan.GetTerrain()->GetBGColorBitmap(), "SceneBG");
-    }
+		// Manipulate DeltaTime
+		if (KeyHeld(KEY_6)) { g_TimerMan.SetDeltaTimeSecs(g_TimerMan.GetDeltaTimeSecs() + 0.001F); }
+		if (KeyHeld(KEY_5) && g_TimerMan.GetDeltaTimeSecs() > 0) { g_TimerMan.SetDeltaTimeSecs(g_TimerMan.GetDeltaTimeSecs() - 0.001F); }
+	}
 
-	// Dump entire map, Ctrl+W
-    if ((key_shifts & KB_CTRL_FLAG) && KeyHeld(KEY_W))
-    {
-        g_FrameMan.SaveWorldToBMP("WorldDump");
-    }
+	// Screenshot, Ctrl+S
+	if (((key_shifts & KB_CTRL_FLAG) && KeyHeld(KEY_S)) || KeyHeld(KEY_PRTSCR)) { g_FrameMan.SaveScreenToBMP("ScreenDump"); }
 
-    // Material draw toggle, Ctrl + M
-    if ((key_shifts & KB_CTRL_FLAG) && KeyPressed(KEY_M))
-        g_SceneMan.SetLayerDrawMode((g_SceneMan.GetLayerDrawMode() + 1) % 3);
+	// Screenshot entire scene, Ctrl+W
+	if ((key_shifts & KB_CTRL_FLAG) && KeyHeld(KEY_W)) { g_FrameMan.SaveWorldToBMP("WorldDump"); }
 
-    // Perf stats display toggle
-    if ((key_shifts & KB_CTRL_FLAG) && KeyPressed(KEY_P))
-		g_PerformanceMan.ShowPerformanceStats(!g_PerformanceMan.IsShowingPerformanceStats());
+	// Dump scene preview (scaled down WorldDump), Alt+W
```

Comment:
Ctrl+P is toggle stats.
I think it's ok to use Alt because this isn't a thing you're gonna be spamming all day and all night.
Seldom used key for a seldom used feature.

Reply:
Hmm, fair point about seldom use, okay, I'm cool with it.